src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jv_jdbc_intro";
+    private static final Properties DB_Properties;
```

Comment:
The variable name `DB_Properties` should be `DB_PROPERTIES` to follow Java naming conventions for constants. Constants in Java are typically written in uppercase letters with underscores separating words.